sdk/appconfiguration/app-configuration/tsconfig.json

```diff
@@ -1,21 +1,11 @@
 {
+  "extends": "../../../tsconfig.package",
   "compilerOptions": {
-    "target": "es6",
-    "module": "es6",
-    "moduleResolution": "node",
-    "strict": true,
-    "sourceMap": true,
-    "inlineSources": true,
-    "declarationMap": true,
-    "declaration": true,
+    "target": "es5",
```

Comment:
the change to target es5 is intentional, yes?

Reply:
I didn't realize app-config needed IE11 support... /cc @richardpark-msft

Reply:
It's not! I was more focused on making the build work, so I got distracted. I'll clean this up.